routing/route/route.go

```diff
@@ -773,3 +773,19 @@ func (r *Route) String() string {
 		b.String(), r.TotalTimeLock,
 	)
 }
+
+// ChanIDString returns the route's channel IDs as a formatted string.
+func (r *Route) ChanIDString() string {
+	var b strings.Builder
```

Comment:
this is very OOP style. I think a helper like `func ChanIDString(r *Route) string` makes a bit more sense since no internal state of Route is being accessed here.